boltdesignsystem/bolt
# DS-671 Add rel="noopener"

## Jira

https://pegadigitalit.atlassian.net/browse/DS-671

## Summary

The attribute `rel="noopener"` was added to the `twig` demos and` js` files

## Details

The attribute `rel="noopener"` was added to:

- twig includes
- link demos
- js files
- test files


 :small_orange_diamond: `Bolt-trigger` and `bolt-chip` were updated due to adding `rel` prop to the schema and then updating the templates and demos. When we check the `bolt-trigger` and `bolt-chip` schemas we can see that they don't use `object attributes` to provide a `target` attribute but there is a custom prop target - so I follow the same way for the `rel` attr.

## How to test

Pull the branch. Check the codebase if `rel="noopener"` is added where should be
